compiler-rt/lib/lsan/lsan_flags.inc

```diff
@@ -43,6 +43,8 @@ LSAN_FLAG(bool, use_poisoned, false,
           "Consider pointers found in poisoned memory to be valid.")
 LSAN_FLAG(bool, log_pointers, false, "Debug logging")
 LSAN_FLAG(bool, log_threads, false, "Debug logging")
+LSAN_FLAG(int, retries, 1,
```

Comment:
Isn't this `tries`? Should we use `1 + retries` in teh code to make sure we never not check?